documentation/help-documenting.rst

```diff
@@ -84,4 +84,12 @@ a pull request directly. It's best to avoid filing a single issue for an entire
 section containing multiple problems; instead, file several issues so that it
 is easier to break the work up for multiple people and more efficient review.
 
+For help with the finer points of English technical writing, mention the
+`@python/proofreaders <https://github.com/orgs/python/teams/proofreaders>`__
+team in your issue or pull request in any `@python <https://github.com/python/>`__
+repo. If you'd like to join the team,
+`open a core-workflow issue <https://github.com/python/core-workflow/issues/new/choose>`__
+similar to
+`python/core-workflow#461 <https://github.com/python/core-workflow/issues/461>`__.
```

Comment:
Instead of linking this specific issue, we should create a template like the [one for triagers](https://github.com/python/core-workflow/issues/new?assignees=&labels=&projects=&template=triage_membership.md&title=Request+for+Python+triage+membership%3A+name) and link to that.

Reply:
Good idea, would you like to create it? :)

Reply:
Sure: 
* https://github.com/python/core-workflow/pull/514